### ISSUE TITLE
Include UIMA types in JavaDoc

### ISSUE THREAD

**Issue by reckart** [dkpro/dkpro-core#496]
```
Currently the UIMA types are excluded from JavaDoc because of the annoying _Type files
that JCasGen creates. As soon as Maven allows to exclude these files from JavaDoc,
we can add the UIMA types to JavaDoc as well.

See also: https://jira.codehaus.org/browse/MJAVADOC-365
```

Original issue reported on code.google.com by `richard.eckart` on 2014-10-08 22:28:59

**Comment by reckart** [comment]
Fixed in master. Needs merge to 1.7.x branch.

**Comment by reckart** [comment]
Merged into 1.7.x.
